.changeset/mighty-mails-design.md

```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/design-system-demo': patch
+---
+
+added the Attribute type="button" to the datepicker to prevent submiting a form on click when the datepicker is directly integrated as shown on the demo page.
```

Comment:
```suggestion
Added the attribute type="button" to the datepicker to prevent submitting a form on click when the datepicker is directly integrated as shown on the demo page.
```